gameplay/src/PlatformLinux.cpp

```diff
@@ -1333,6 +1334,13 @@ int Platform::enterMessagePump()
             _game->frame();
         }
 
+        if (!visible && __vsync) {
+        	// glXSwapBuffers may not block on vsync properly if the window is obscured/offdesktop --
+        	// if vsync enabled, do an artificial slowdown - 15ms is largely arbitrary
+        	// if the we have disabled vsync, we don't care, run full speed anyway.
+        	usleep(15000.0);
```

Comment:
I doubt this is the right way to do vsync. The purpose of vsync is not in slowdown, but in fixed frame rate and to ensure that new frame is not being presented when the last one is still drawing.